stdlib/2and3/socket.pyi

```diff
@@ -539,7 +539,7 @@ class socket:
     def gettimeout(self) -> float: ...
     def ioctl(self, control: object,
               option: Tuple[int, int, int]) -> None: ...
-    def listen(self, backlog: int) -> None: ...
+    def listen(self, backlog: int = ...) -> None: ...
```

Comment:
The `backlog` parameter is optional from Python 3.5 onwards. Since we still support Python 3.4 we need something like this:
```suggestion
    if sys.version_info < (3, 5):
        def listen(self, backlog: int) -> None: ...
    else:
        def listen(self, backlog: int = ...) -> None: ...
```